onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -318,9 +320,9 @@ TEST(GraphTransformationTests, FuseConvBnAddMulFloat16) {
   std::unique_ptr<ConvBNFusion> ConvBNFusion_transformer = std::make_unique<ConvBNFusion>();
   std::unique_ptr<ConvMulFusion> ConvMulFusion_transformer = std::make_unique<ConvMulFusion>();
   std::unique_ptr<ConvAddFusion> ConvAddFusion_transformer = std::make_unique<ConvAddFusion>();
-  session_object.RegisterGraphTransformer(std::move(ConvBNFusion_transformer));
-  session_object.RegisterGraphTransformer(std::move(ConvMulFusion_transformer));
-  session_object.RegisterGraphTransformer(std::move(ConvAddFusion_transformer));
+  session_object.RegisterGraphTransformer(std::move(ConvBNFusion_transformer), {onnxruntime::kCpuExecutionProvider});
+  session_object.RegisterGraphTransformer(std::move(ConvMulFusion_transformer), {onnxruntime::kCpuExecutionProvider});
+  session_object.RegisterGraphTransformer(std::move(ConvAddFusion_transformer), {onnxruntime::kCpuExecutionProvider});
 
```

Comment:
As we discussed offline, actually, Conv-BN, Conv-Mul and Conv-Add should not be CPU execution provider specific.

Reply:
In this PR I am not enabling them... lets revisit this in next PR when I enable all transformers....